build.gradle

```diff
@@ -4,7 +4,8 @@ import java.util.regex.Pattern
 plugins {
     id 'net.neoforged.gradleutils' version '3.0.0-alpha.10' apply false
     id 'com.diffplug.spotless' version '6.22.0' apply false
-    id 'net.neoforged.licenser' version '0.7.2' apply false
+    id 'net.neoforged.licenser' version '0.7.2'
```

Comment:
Is licenser still running on all subprojects correctly?

Reply:
Yes, the apply false was useless since we'd apply it just under.